KubeArmor/core/kubeUpdate.go

```diff
@@ -48,19 +48,37 @@ func (dm *KubeArmorDaemon) HandleNodeAnnotations(node *tp.Node) {
 		node.Annotations["kubearmor-policy"] = "enabled"
 	}
 
-	if lsm, err := os.ReadFile("/sys/kernel/security/lsm"); err == nil {
-		if !strings.Contains(string(lsm), "apparmor") && !strings.Contains(string(lsm), "selinux") {
-			// exception: neither AppArmor nor SELinux
-			if node.Annotations["kubearmor-policy"] == "enabled" {
-				node.Annotations["kubearmor-policy"] = "audited"
-			}
+	// == LSM == //
+	var lsm string
+
+	// Check if enforcer is set in the node annotations
+	if v, ok := node.Annotations["kubearmor.io/enforcer"]; ok {
+		lsm = v
+	} else { // Read the lsm from the system
+		lsmByteData, err := os.ReadFile("/sys/kernel/security/lsm")
+		if err != nil {
+			kg.Errf("Failed to read /sys/kernel/security/lsm (%s)", err)
```

Comment:
Minor changes based on operator changes introduced in the latest release.
```suggestion
	if v, ok := node.Labels["kubearmor.io/enforcer"]; ok {
		lsm = v
	} else { // Read the lsm from the system
		lsmByteData, err := os.ReadFile("/sys/kernel/security/lsm")
		if err != nil && !os.IsNotExist(err) {
			kg.Errf("Failed to read /sys/kernel/security/lsm (%s)", err.Error())
```